e2e/android/app/src/main/java/com/e2eoktareactnative/generated/BasePackageList.java

```diff
@@ -0,0 +1,21 @@
+package com.e2eoktareactnative.generated;
```

Comment:
Should this file be checked in? I only ask because it's in a generated package/folder.

Reply:
Probably it's better to add it to .gitignore